### ISSUE TITLE
Custom collections do not have any data

### ISSUE THREAD

**Issue by tannerdolby** [colabottles/dgc#3]
Currently custom collections don't have any data because `/src/breakfast/` + `/src/lunch/` + `/src/miscellaneous/` don't contain any Markdown files. 

@toddlibby / @colabottles 

Current issues: 

1. The custom collections in `.eleventy.js` currently serves no purpose because the method `.getFilteredByGlob()` can't match a glob since the `/breakfast/` and `/lunch/` directories don't exist and do not contain any markdown files.
2. Unless we create Markdown files within the respective directories. Something like, `/breakfast/wrap-one.md`, `/breakfast/wrap-two.md/` with the menu item data as Front matter to be accessed within the custom collection. That would be the only way these custom collections could be useful for generating menu item data.

```
// Returns breakfast items, sorted by display order
  config.addCollection('breakfast', collection => {
    return sortByDisplayOrder(collection.getFilteredByGlob('./src/breakfast/*.md'));
  });
  // Returns lunch items, sorted by display order
  config.addCollection('lunch', collection => {
    return sortByDisplayOrder(collection.getFilteredByGlob('./src/lunch/*.md'));
  });
  // Returns breakfast items, sorted by display order
  config.addCollection('miscellaneous', collection => {
    return sortByDisplayOrder(collection.getFilteredByGlob('./src/miscellaneous/*.md'));
  });

```

**Comment by tannerdolby** [comment]
## Proposed Fix

I was able to populate some of the menu data by creating my a separate global data file `test.json` with an array of objects for the "Breakfast" section. This way there is not any extra object keys and we can simply iterate over the data with:

 ```
{% for item in test %} {{ item.name }} {% endfor %}
```
Filename: `_data/test.json`
```
[
    {
        "name": "Egg Simple",
        "description": "One (1) egg your way with American cheese in a delicious wrap.",
        "prepared": "Grilled",
        "price": "$4.25",
        "allergens": ["eggs", "dairy"],
        "id": 1
    },
...
```
If the markdown files are created to be used in a custom collection like mentioned in my comment above, they will most likely only contain front matter data. Custom collections work best with blog posts or markdown files that have `{{ content }}` and not only front matter. 

> This is where generating page content from global data is probably more fitting.

I think that creating a global data file with the menu item content for Breakfast, Lunch and misc will be a better option than creating a whole bunch of `.md` files that contain the menu item content within Front Matter.

**Comment by colabottles** [comment]
Sounds good. I can deal with that.

**Comment by tannerdolby** [comment]
Proposed fix in #4
